### ISSUE TITLE
Advisory Technique as Success Criteria (G201)

### ISSUE THREAD

**Issue by AutoSponge** [a11yproject/a11yproject.com#927]
ref: https://a11yproject.com/checklist/#identify-links-that-open-in-a-new-window

From the Introduction
> Success criteria
> Each item on this checklist has a corresponding WCAG "success criterion."

This section only references the advisory technique G201. It's not required for conformance (although I wish it were) so something may need to change to make it clear that this is (for now) a best practice.

**Comment by ericwbailey** [comment]
Oh hm, you're right. How does this sound to you?

>Doing this will help people understand what will happen when activating the link. Ideally this information should be communicated in a way that is available to all users. While this technique is not required for compliance, it is an often-cited area of frustration for many different kinds of assistive technology users.

**Comment by AutoSponge** [comment]
LGTM 👍

**Comment by ericwbailey** [comment]
Closed via https://github.com/a11yproject/a11yproject.com/commit/2d63ee5c02b6445937fa7bed013f2cf3613e3ef5. Thank you!
